frontend/app/routes/protected/stub-login.tsx

```diff
@@ -1,156 +0,0 @@
-
-export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
-  if (!data) return [];
-  return getTitleMetaTags(data.meta.title);
-});
-
-export async function loader({ context: { session }, request }: LoaderFunctionArgs) {
-  featureEnabled('stub-login');
-
-  const t = await getFixedT(request, handle.i18nNamespaces);
-  const meta = { title: t('gcweb:meta.title.template', { title: t('stub-login:index.page-title') }) };
-
-  const idToken: IdToken | undefined = session.get('idToken');
-  const userInfoToken: UserinfoToken | undefined = session.get('userInfoToken');
-
-  const defaultValues = {
-    sin: userInfoToken?.sin ?? '',
-    sid: idToken?.sid ?? '00000000-0000-0000-0000-000000000000',
-    sub: idToken?.sub ?? '00000000-0000-0000-0000-000000000000',
-  };
-
-  return { meta, defaultValues };
-}
-
-export async function action({ context: { session }, params, request }: ActionFunctionArgs) {
-  featureEnabled('stub-login');
-
-  const t = await getFixedT(request, handle.i18nNamespaces);
-
-  const stubLoginSchema = z.object({
-    sin: z.string().trim().min(1, t('stub-login:index.error-message.sin-required')),
-    sid: z.string().trim().min(1, t('stub-login:index.error-message.sid-required')),
-    sub: z.string().trim().min(1, t('stub-login:index.error-message.sub-required')),
-  });
-
-  const formData = await request.formData();
-
-  const data = {
-    sin: String(formData.get('sin') ?? ''),
-    sid: String(formData.get('sid') ?? ''),
-    sub: String(formData.get('sub') ?? ''),
-  };
-
-  const parsedDataResult = stubLoginSchema.safeParse(data);
-
-  if (!parsedDataResult.success) {
-    return json({
-      errors: transformFlattenedError(parsedDataResult.error.flatten()),
-    });
-  }
-
-  const sin = parsedDataResult.data.sin;
-  const sid = parsedDataResult.data.sid;
-  const sub = parsedDataResult.data.sub;
-
-  const currentDateInSeconds = Math.floor(UTCDate.now() / 1000);
-
-  const idToken = {
-    iss: 'GC-ECAS',
-    jti: '71b080e9-2524-4572-a085-a53e63a98116',
-    nbf: currentDateInSeconds - 30,
-    exp: currentDateInSeconds + 300, //five minutes TTL for the token
-    iat: currentDateInSeconds,
-    aud: 'CDCP',
-    sub,
-    nonce: 'hqwVxGbvJ5g7NSWoOv1BvrA9avVAY7CL',
-    locale: 'en-CA',
-  };
-
-  const userinfoTokenPayload = {
-    aud: 'CDCP',
-    birthdate: '2000-01-01',
-    iss: 'GC-ECAS-MOCK',
-    locale: 'en-CA',
-    sid,
-    sin,
-    sub,
-    mocked: true,
-  };
-  const userInfoToken: UserinfoToken = session.get('userInfoToken');
-  if (!session.has('userInfoToken')) {
-    session.set('userInfoToken', userinfoTokenPayload);
-  } else {
-    userInfoToken.sin = sin;
-    userInfoToken.sub = sub;
-    session.set('userInfoToken', userInfoToken);
-  }
-  session.set('idToken', idToken);
-
-  const subscriptionService = getSubscriptionService();
-  const user = await subscriptionService.getUserByRaoidcUserId(session.get('userInfoToken').sub);
-  session.set('userId', user?.id);
```

Comment:
The `stub-login` (including its feature flag and i18n files) route can still be kept as it is for impersonating a user within MSCA. However, we can remove this service call to retrieve the `user` that is used for alerts/subscriptions.